docker/convert.go

```diff
@@ -56,11 +56,13 @@ func ConvertToAPI(s *Service) (*ConfigWrapper, error) {
 func volumes(c *project.ServiceConfig, ctx project.Context) map[string]struct{} {
 	volumes := make(map[string]struct{}, len(c.Volumes))
 	for k, v := range c.Volumes {
-		vol := ctx.ResourceLookup.ResolvePath(v, ctx.ComposeFiles[0])
+		if len(ctx.ComposeFiles) > 0 {
```

Comment:
Would a sole presence of compose file in the context mean that the volume path is relative? What if the file is present, but the volume path is absolute?